websites/B/BigBlueButton/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+        "name": "Oranż Metylowy",
+        "id": "359982590972788737"
+    },
+    "url": "demo.bigbluebutton.org",
+    "regExp": "\\/b\\/[\\w]{3}-[\\d\\w]{3}-[\\d\\w]{3}|html5client\\/join\\?",
+    "description": {
```

Comment:
for what exactly is this regex?

Reply:
This is for checking the meeting URL, as BBB is mostly self-hosted so a universal domain URL is not really possible.

Reply:
Could you provide an example url for me.

Reply:
https://bbb.school.city.com/b/xyz-97h-jki
or after joining the session
https://bbb.school.city.com/html5client/join?sessionToken=1234567890abcdef
Will update the regexp to also match the guest lobby URLs in the next commit.

Reply:
`[/]b[/][\w]{3}-[\d\w]{3}-[\d\w]{3}|html5client[/]join[?]`
![image](https://user-images.githubusercontent.com/34792838/96656133-7911a900-133f-11eb-9692-92a19f609b90.png)
Make sure you use the JS version cause some stuff wasn't working.

Reply:
![obraz](https://user-images.githubusercontent.com/30433568/96693918-17c1f800-1388-11eb-926b-c83300b219d8.png)
Sure, the regex works fine on both PCRE and JS implementations. The PreMiD extension also handles it correctly (thus showing correct presence, as in my first screenshot).

Reply:
We would prefer it if you use the JS flavor since its a JS script.
And instead of escaping with backslash put [] around it.

Reply:
Okay, done.